src/App/Page.php

```diff
@@ -253,9 +253,9 @@ private function initHead(App $app, Arguments $args, L10n $l10n, IManageConfigVa
 			'$touch_icon'      => $touch_icon,
 			'$block_public'    => intval($config->get('system', 'block_public')),
 			'$stylesheets'     => $this->stylesheets,
-			'$likeError'       => $l10n->t('Like not successfull'),
-			'$dislikeError'    => $l10n->t('Dislike not successfull'),
-			'$announceError'   => $l10n->t('Sharing not successfull'),
+			'$likeError'       => $l10n->t('Like not successful'),
+			'$dislikeError'    => $l10n->t('Dislike not successful'),
+			'$announceError'   => $l10n->t('Sharing not successful'),
```

Comment:
As noted, these don't appear to have hit Transifex, so I can't fix them there and no one else could have translated them yet...

Reply:
You are right, the import to Transifex is manual, thanks again!